src/python/xclbin_binding.py

```diff
@@ -83,7 +178,7 @@ class mem_data (ctypes.Structure):
     _fields_ = [
         ("m_type", ctypes.c_uint8),
         ("m_used", ctypes.c_uint8),
-        ("mem_u1", mem_u1),
+        ("mem_u1", m_u1),
```

Comment:
What's the reason for changing this name? If this should be 'm_u1' should the next line be 'm_u2'?